IT/src/test/java/module-info.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/*
+ Module for the purposes of housing tests that need to be in a module in order to run.
+ */
+open module com.microsoft.gctoolkit.integration {
```

Comment:
`open` might not be needed here. We tried that out, but it didn't solve the test runtime issue.